mindmeld/ser.py

```diff
@@ -63,33 +38,17 @@ def get_candidates(
     Returns:
         list of QueryEntity: The system entities found in the query
     """
-    dims = _dimensions_from_entity_types(entity_types)
-    language = language or query.language
-    time_zone = time_zone or query.time_zone
-    timestamp = timestamp or query.timestamp
-    response, response_code = parse_numerics(
-        query.text,
-        dimensions=dims,
+    logger.warning(
+        "get_candidates is deprecated in favor of DucklingRecognizer.get_candidates."
+    )
+    DucklingRecognizer.get_instance().get_candidates(
```

Comment:
Why does this file only use `DucklingRecognizer ` while the other files can use `NoOp`, `Duckling` or anything else?

Reply:
Nmv looks like this is used just for deprecation. Do we have to do `set_system_entity_recognizer` before this call though?

Reply:
Since the deprecated method uses Duckling, `Duckling.get_instance() `should return a `DucklingRecognizer` even if `SystemEntityRecognizer.get_instance()` returns a `NoOp`